README.md

```diff
@@ -80,6 +80,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Gurkhatech](https://gurkhatech.com/) | New Baneshwor, Kathmandu | App & Web Development, Facebook Marketing Nepal
 - [Heartsun Technology](http://heartsunntsoft.com/) | Bharatpur, Chitwan | Software Company with .NET platform
 - [HelloWorldCorp](http://helloworldcorp.biz/) | Gairidhara, Kathmandu | Web Development Company
+- [Hiup Solutions Pvt. Ltd.](https://olivelearning.com/) | Dhumbarahi, Kathmandu | Web Development Company
```

Comment:
@realanmup Thank you for the pull request. Can you update the description of the company? `Web Development Company` is a bit general.

Is the location correct? I see that the website has `Chabahil, Kathmandu`

Reply:
@mesaugat I've updated the description and detailed location now.